app/assets/javascripts/views/editor/editor_view.ts

```diff
@@ -261,14 +264,63 @@ class EditorViewCtrl extends PureViewCtrl<unknown, EditorState> {
           desc: 'Changes not saved',
         });
         break;
+      case ApplicationEvent.ProtectionSessionExpiryDateChanged: {
+        if (!this.note.protected) {
+          return;
+        }
+        if (data.protectionDuration === ProtectionSessionDurations[0].valueInSeconds) {
+          await this.setTimerForNoteProtection(DURATION_TO_POSTPONE_PROTECTED_NOTE_LOCK_WHILE_EDITING);
+          // TODO: this is required when we select the protected note for the first time and set "Don't remember" -
```

Comment:
Are these tests going to be included in this PR?

Reply:
No, everything will be cleaned up :)